cart-service/serverless.yml

```diff
@@ -0,0 +1,25 @@
+service: cart-service
+
+plugins:
+  - serverless-offline
+  - serverless-dotenv-plugin
```

Comment:
According to the serverless documentation, there's no need to use this plugin anymore:
https://www.serverless.com/plugins/serverless-dotenv-plugin#do-you-need-this-plugin

Reply:
As you can see I do not duplicate env names in serverless.yaml file
plugin is do the following: "This plugin is only useful if you want to automatically import all variables from .env into functions:"